packages/website/pages/[...slug].tsx

```diff
@@ -40,12 +39,10 @@ export default function ComponentPage(props: ComponentPageProps) {
       <Head>
         <title>Forma 36 - {props.frontMatter.title}</title>
       </Head>
-
       <article className={styles.root}>
-        <Link href="/">Back</Link>
         <h1>{props.frontMatter.title}</h1>
         <div>
-          <MDXRemote {...props.source} />
+          <MdxRenderer source={props.source} />
         </div>
```

Comment:
I aproved already but I forgot that we need to change this

We need to remove this `div` wrapping the MDXRemote/Renderer

And the root styles needs to be:
```
const styles = {
  root: css({
    display: 'flex',
    flexDirection: 'column',
  }),
};
```

Or the content will overlapp the footer of the website

Reply:
@gui-santos I'll do this in the follow-up PR, thanks for the suggestion